packages/wallet/src/persistence/pouchdbStores/pouchdbWalletStores.ts

```diff
@@ -90,6 +93,7 @@ export const createPouchdbWalletStores = (
       logger
     ),
     unspendableUtxo: new PouchdbUtxoStore({ dbName: `${baseDbName}UnspendableUtxo` }, logger),
-    utxo: new PouchdbUtxoStore({ dbName: `${baseDbName}Utxo` }, logger)
+    utxo: new PouchdbUtxoStore({ dbName: `${baseDbName}Utxo` }, logger),
+    volatileTransactions: new PouchdbConfirmedTransactionsStore(docsDbName, 'confirmedTransactions', logger)
```

Comment:
```suggestion
    volatileTransactions: new PouchdbConfirmedTransactionsStore(docsDbName, 'volatileTransactions', logger)
```

Reply:
Done in: [ddb6af8](https://github.com/input-output-hk/cardano-js-sdk/pull/351/commits/ddb6af800a591149f87a2668b4da04d0470a29c7)